src/unstract/sdk/x2txt.py

```diff
@@ -68,6 +69,16 @@ def process(
         output_file_path: Optional[str] = None,
         **kwargs: dict[Any, Any],
     ) -> str:
+        if kwargs.get("enable_highlight", False) and isinstance(
```

Comment:
@johnyrahul I suggest renaming `enable_highlight` argument instead to something generic like `append_metadata` / `provide_metadata`. Later on for other providers we can try to include support based on what they provide.